tutorials/navigation/navigation_using_navigationmeshes.rst

```diff
@@ -200,6 +200,51 @@ The geometry data should be in general kept very simple. As many edges as are re
 Especially in 2D duplicated and nested geometry should be avoided as it forces polygon hole calculation that can result in flipped polygons.
 An example for nested geometry would be a smaller StaticBody2D shape placed completely inside the bounds of another StaticBody2D shape.
 
+Baking navigation mesh chunks for large worlds
+----------------------------------------------
+
+.. figure:: img/navmesh_chunk_build.gif
+   :align: center
+   :alt: Building navigation mesh chunks
+
+   Building and updating individual navigation mesh chunks at runtime.
+
+To avoid misaligned edges between different region chunks the navigation meshes have two important properties for the navigation mesh baking process. The baking bound and the border size. Together they can be used to have perfectly aligned edges between region chunks.
```

Comment:
```suggestion
To avoid misaligned edges between different region chunks the navigation meshes have two important properties
for the navigation mesh baking process. The baking bound and the border size.
Together they can be used to ensure perfectly aligned edges between region chunks.
```